axelar-chains-config/info/mainnet.json

```diff
@@ -224,7 +224,7 @@
         }
       },
       "explorer": {
-        "url": "https://snowtrace.io",
+        "url": "https://avalanche.routescan.io",
```

Comment:
this just redirects to snowtrace.dev. snowtrace.io still works so I think it's ok to keep that?